concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-303	303	mud, marsh	640	MUD
+RonaTas-2011-431-304	304	yellow	1424	YELLOW
+RonaTas-2011-431-305	305	dragon	2038	DRAGON
+RonaTas-2011-431-306	306	sickle	341	SICKLE
+RonaTas-2011-431-307	307	a type of wild onion, Ornithogalum		
+RonaTas-2011-431-308	308	footwear, sandals		
+RonaTas-2011-431-309	309	tent	1316	TENT
+RonaTas-2011-431-310	310	fast, quick	1631	FAST
+RonaTas-2011-431-311	311	to sweep, to broom	230	SWEEP
+RonaTas-2011-431-312	312	broom	245	BROOM
+RonaTas-2011-431-313	313	draff, lees, dregs of wine		
+RonaTas-2011-431-314	314	to injure, to hurt	44	INJURE
+RonaTas-2011-431-315	315	army, troops, crowd	1561	ARMY
+RonaTas-2011-431-316	316	a nit	267	NIT
+RonaTas-2011-431-317	317	cup, goblet	476	CUP
+RonaTas-2011-431-318	318	bristles, also short sheaves used to strengthen a thatched roof		
+RonaTas-2011-431-319	319	even, flat, open (water)	1633	FLAT
+RonaTas-2011-431-320	320	smooth, simít	1234	SMOOTH
+RonaTas-2011-431-321	321	running water, running channel under a mill		
+RonaTas-2011-431-322	322	spring, well, brook	1537	SPRINGTIME
+RonaTas-2011-431-323	323	much, many	1198	MANY
+RonaTas-2011-431-324	324	falcon	3258	FALCON
+RonaTas-2011-431-325	325	dogwood (tree), cornel, Cornus		
+RonaTas-2011-431-326	326	beer	1639	BEER
+RonaTas-2011-431-327	327	sterlet, sturgeon, Acipenser stellatus, ruthenus	3606	STURGEON
+RonaTas-2011-431-328	328	pike perch, zander, Stizostedion lucioperca		
+RonaTas-2011-431-329	329	scurvy		
+RonaTas-2011-431-330	330	dense, thick (of woods or soup)	2239	DENSE
+RonaTas-2011-431-331	331	brother-in-law	2314	BROTHER-IN-LAW
+RonaTas-2011-431-332	332	mirage		
+RonaTas-2011-431-333	333	uvula, the recurved point of a fishing hook or a lance, red wattle on a chicken	2293	UVULA
+RonaTas-2011-431-334	334	beard	678	BEARD
+RonaTas-2011-431-335	335	raft	1664	RAFT
+RonaTas-2011-431-336	336	number	1356	NUMBER
+RonaTas-2011-431-337	337	to have pity for, to regret	1875	REGRET
+RonaTas-2011-431-338	338	to wish, to intend something for somebody/something, to devote	2117	DESIRE
+RonaTas-2011-431-339	339	trousseau box		
+RonaTas-2011-431-340	340	bucket, wooden pail (with a handle)	2052	BUCKET
+RonaTas-2011-431-341	341	light-coloured, yellowish, pale, (partly) bald	1125	BALD
+RonaTas-2011-431-342	342	wing	1257	WING
+RonaTas-2011-431-343	343	grocer, grand handler		
+RonaTas-2011-431-344	344	chair, seat, bench, throne	1476	CHAIR
+RonaTas-2011-431-345	345	wind	960	WIND
+RonaTas-2011-431-346	346	to slumber, to doze		
+RonaTas-2011-431-347	347	beautiful	1427	BEAUTIFUL
+RonaTas-2011-431-348	348	freckle, sun spot, stain	530	FRECKLE
+RonaTas-2011-431-349	349	part of a village, street, group of people; row, line, order	1362	STREET
+RonaTas-2011-431-350	350	threshing floor (round)		
+RonaTas-2011-431-351	351	fumes, vapour, spirits, alcohol, humours	404	ALCOHOL (FERMENTED DRINK)
+RonaTas-2011-431-352	352	colour, face, external appearance, the best part of something, the upperpart of something	1560	FACE
```

Comment:
I'd unmap due to the variety of meanings

Reply:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534